### PR TITLE
Log collector

### PR DESCRIPTION
**Pull request by vdombrovski**

### CODE REVIEW

**Review by fatpat — CHANGES_REQUESTED** [review]

File: cmd/log.plugin/log.plugin.go [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+	family := "log"
+
+	// Dimensions will be added dynamically
+	responseCode := netdata.NewChart("log", "response_code", "", "Response code", "ops", family, "")
+	worker.AddChart(responseCode, collector)
+
+	responseTime := netdata.NewChart("log", "response_time", "", "Response time", "ms", family, "")
+	worker.AddChart(responseTime, collector)
+
+	bandwidthIn := netdata.NewChart("log", "bandwidth_in", "", "Bandwidth in", "kB", family, "")
```
Comment (fatpat) [inline cmd/log.plugin/log.plugin.go]: bandwidth with a unit of `kB`, should it not be `kBps` or `kbps` ?

**Review by fatpat — CHANGES_REQUESTED** [review]

File: log/log.go [hunk 2]
```diff
@@ -0,0 +1,332 @@
...
+				c.metrics.Lock()
+				if _, ok := c.metrics.Requests[reqName]; ok {
+					c.metrics.Requests[reqName]++
+				} else {
+					c.metrics.Requests[reqName] = 1
+				}
+				c.metrics.Unlock()
+			}
+
+			reqName := s[method] + "_" + name + tagString
```
Comment (fatpat) [inline log/log.go]: in line 252 a dot is used to prefix the name whereas here an underscore is used. Please use dot everywhere to be consistent (better for regex parsing in prometheus relabelling phase)

**Review by fatpat — CHANGES_REQUESTED** [review]
```
BEGIN log.response_time
SET 'min_PUT_OPENIO-oioswift-0.-' = 0
SET 'max_DELETE_OPENIO-rawx-0' = 143
SET 'max_PUT_OPENIO-rawx-0' = 36891
SET 'avg_DELETE_OPENIO-rawx-0' = 0
SET 'avg_M2_CREATE_OPENIO-meta2-0' = 0
SET 'avg_PUT_OPENIO-rawx-0' = 0
SET 'avg_M1_LINK_OPENIO-meta1-0' = 0
SET 'min_M2_PUT_OPENIO-meta2-0' = 0
SET 'max_M2_PUT_OPENIO-meta2-0' = 140344
SET 'min_M1_LINK_OPENIO-meta1-0' = 0
SET 'max_M1_LIST_OPENIO-meta1-0' = 28969
SET 'max_M2_CREATE_OPENIO-meta2-0' = 275720
SET 'avg_PUT_OPENIO-oioswift-0.-' = 0
SET 'min_M1_LIST_OPENIO-meta1-0' = 0
SET 'avg_M1_LIST_OPENIO-meta1-0' = 0
SET 'avg_PUT_OPENIO-oioswift-0.put-object' = 0
SET 'avg_M2_PUT_OPENIO-meta2-0' = 0
SET 'min_M2_CREATE_OPENIO-meta2-0' = 0
SET 'min_DB_PGET_OPENIO-meta2-0' = 0
SET 'min_DELETE_OPENIO-rawx-0' = 0
SET 'max_PUT_OPENIO-oioswift-0.put-object' = 10.0754
SET 'avg_DB_PGET_OPENIO-meta2-0' = 0
SET 'min_PUT_OPENIO-oioswift-0.put-object' = 0
SET 'max_PUT_OPENIO-oioswift-0.-' = 10.0071
SET 'max_M1_LINK_OPENIO-meta1-0' = 220
SET 'max_DB_PGET_OPENIO-meta2-0' = 196895
SET 'min_PUT_OPENIO-rawx-0' = 0
```

**Comment by cdelgehier** [comment]
Can we close this one ? @fatpat

**Pull request closed without merging**